crates/sui/src/main.rs

```diff
@@ -45,12 +45,14 @@ async fn main() {
     let bin_name = env!("CARGO_BIN_NAME");
     let args = Args::parse();
     let _guard = match args.command {
-        SuiCommand::Console { .. } | SuiCommand::Client { .. } => {
-            telemetry_subscribers::TelemetryConfig::new()
-                .with_log_file(&format!("{bin_name}.log"))
-                .with_env()
-                .init()
-        }
+        SuiCommand::Console { .. }
+        | SuiCommand::Client { .. }
+        | SuiCommand::KeyTool { .. }
+        | SuiCommand::Move { .. } => telemetry_subscribers::TelemetryConfig::new()
+            .with_log_file(&format!("{bin_name}.log"))
```

Comment:
@patrickkuo, looking at the history (which is kind of ancient now), you were the last person to touch the code responsible for adding the use of a log file here.  Do you remember why these commands needed their default behaviour changed?  Given there is an environment variable for this (`RUST_LOG_FILE`) it would seem more straightforward just to leave this to the user.

Reply:
ya that's ancient lol, I cannot recall why we do it this way, I think we added the log file everywhere when we first introduce telemetry....

Reply:
Okay, if no one is opposed, I would vote for removing this tweak, and instructing people who want this behaviour to set the log file themselves using the environment variable.

I half suspect that it exists because the logs were showing up when they shouldn't be, and this change made that less disruptive, but if that's the case, we should just invest the time to understand and fix that issue.

Reply:
I agree that we should just remove the log file default, today it just clutters user's directories after using a sui command

Reply:
Sounds good!